maxim/logger/livekit/agent_session.py

```
@@ -202,9 +233,113 @@ def handle_tool_call_executed(self, event: FunctionToolsExecutedEvent):
                 tool_output = output.output
                 break
         tool_call.result(tool_output)
+        
+
+def handle_agent_response_complete(self: AgentSession, response_text):
```

Ruff (0.12.2)

`handle_agent_response_complete` is too complex (12 > 10)

(C901)